gulp/tasks/pl_scripts.js

```diff
@@ -11,7 +11,9 @@ gulp.task('pl_scripts', ['pl_scripts-lint'], function() {
             './_src/pl/vendor/jquery/jquery-2.1.3.min.js',
             './_src/pl/vendor/jquery/jquery.smooth-scroll.js',
             config.pl_src + '/ui.js',
-            config.pl_src + '/color-contrast.js'
+            config.pl_src + '/size-slider.js',
+            config.pl_src + '/color-contrast.js',
+            './_src/vendor/svg4everybody/svg4everybody.min.js' // polyfill for SVGs in IE9-11
```

Comment:
This is a script that is associated with the icons themselves but this task and JS aggregated is for the PL documentation site. 

Does it make sense to keep this here or to move it to a new scripts task and separately added ``<script src="{{ site.baseurl }}public/js/main.min.js"></script>`` asset to our pages?